src/arguments/CoreNumber.ts

```diff
@@ -1,41 +1,27 @@
 import type { PieceContext } from '@sapphire/pieces';
 import { Identifiers } from '../lib/errors/Identifiers';
+import { resolveNumber } from '../lib/resolvers';
 import { Argument, ArgumentContext, ArgumentResult } from '../lib/structures/Argument';
 
 export class CoreArgument extends Argument<number> {
+	private readonly messages = {
+		[Identifiers.ArgumentNumberTooSmall]: ({ minimum }: ArgumentContext) => `The argument must be greater than ${minimum}.`,
+		[Identifiers.ArgumentNumberTooBig]: ({ maximum }: ArgumentContext) => `The argument must be less than ${maximum}.`,
+		[Identifiers.ArgumentNumberError]: () => 'The argument did not resolve to a valid number.'
```

Comment:
```suggestion
		[Identifiers.ArgumentNummberTooSmall]: ({ minimum }: ArgumentContext) => `The given number must be greater than ${minimum}.`,
		[Identifiers.ArgumentNumberTooLarge]: ({ maximum }: ArgumentContext) => `The given number must be less than ${maximum}.`,
		[Identifiers.ArgumentNumberError]: () => 'The argument did not resolve to a valid number.'
```

And update `resolveNumber` accordingly